app/blueprints/activity_execution_blueprint.rb

```diff
@@ -14,4 +14,12 @@ class ActivityExecutionBlueprint < Blueprinter::Base
   field :languages do |activity_execution|
     available_languages_for_frontend(activity_execution)
   end
+
```

Comment:
maybe we could improve performance by not including field and spot association on all of them?

Reply:
Finde ich zum jetzigen Zeitpunkt nicht nötig, ausser es gäbe jetzt bereits Performance-Probleme?